roles/openshift_health_checker/openshift_checks/etcd_volume_check.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about etcd volume size.
+"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdVolumeCheck(OpenShiftCheck):
+    """Check that etcd volume capacity is not over 50%"""
+
+    name = "etcd_volume_check"
+    tags = ["etcd_volume_check", "etcd"]
```

Comment:
- A tag should be something more generic that would describe a group of checks, let's no include the name of the check as a tag.
- `etcd` is good, we may have more checks related to etcd :+1: 

When is this check useful to be run? We could think of a tag based on that.
@sosiouxme suggestions?